sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -605,18 +606,12 @@ static void _send(gnrc_pktsnip_t *pkt, bool prep_hdr)
     hdr = ipv6->data;
     payload = ipv6->next;
 
-    gnrc_netif_t *netif = (iface == KERNEL_PID_UNDEF) ?
-                            NULL :
-                            gnrc_netif_get_by_pid(iface);
     if (ipv6_addr_is_multicast(&hdr->dst)) {
         _send_multicast(netif, pkt, ipv6, payload, prep_hdr);
     }
-    else if ((ipv6_addr_is_loopback(&hdr->dst)) ||      /* dst is loopback address */
-             ((iface == KERNEL_PID_UNDEF) && /* or dst registered to any local interface */
```

Comment:
There is no equivalent to this line, that's why so `netif` pre-selected above is overwritten in the in the line below.